### PR TITLE
Move redirects from global.conf to app.

### PR DESCRIPTION
**Pull request by pmclanahan**
This handles most of the redirects in global.conf.
Some remain, but this is now to the point where
the test_redirects suite and mcom-tests passes against a demo
server running Apache.

### CODE REVIEW

**Comment by pmclanahan** [comment]
@jgmize it has arrived. r?

**Comment by jgmize** [comment]
@pmclanahan I think this is good except for the flake8 errors travis is complaining about. r+wc

**Comment by jgmize** [comment]
This is nice work, very thorough, great attention to detail. I really appreciate you slogging through all this.

**Comment by pmclanahan** [comment]
Thanks. I think "slog" is a good word for it ;)

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -290,7 +290,7 @@ def dnt(request):
 def all_downloads(request, channel):
     if channel is None:
         channel = 'release'
-    if channel == 'developer':
+    if channel == 'developer' or channel == 'aurora':
```
Comment (jgmize) [inline bedrock/firefox/views.py]: Nice catch.

**Pull request merged**